source/_patterns/20-molecules/teaser/teaser.scss

```diff
@@ -891,7 +891,6 @@
   align-items: stretch;
   flex-direction: column;
   justify-content: flex-end;
-  max-height: 100%;
```

Comment:
Muss das wirklich raus? Das fixed (sollte eigentlich) ein Flexbox-Grid-Bug in Safari 🤔

Reply:
Würde ich mir sonst nochmals anschauen, sobald Test wieder geht 😉